invenio/modules/authors/recordext/fields/author_base.cfg

```diff
@@ -65,7 +65,7 @@ name:
          'schema': {'last': {'type': 'string', 'empty': False},
                     'first': {'type': 'string', 'empty': False}}}}
     creator:
-        marc, "100__", {'last': util_split(value['a'], ', ', 0),
+        marc, "100._", {'last': util_split(value['a'], ', ', 0),
```

Comment:
@kaplun the values that you can put in there are python regular expressions, so you can put something like `100[013]_` if you want to restrict the first indicator to some values.